bridge/src/config.rs

```diff
@@ -18,6 +18,7 @@ pub struct Config {
 	pub foreign: Node,
 	pub authorities: Authorities,
 	pub txs: Transactions,
+	pub estimated_gas_cost_of_withdraw: u32,
```

Comment:
please also update `examples/config.json` and `README.md`

Reply:
done